scripts/embed_question.py

```diff
@@ -0,0 +1,35 @@
+import sys
+
+import typer
+from prediction_market_agent_tooling.markets.agent_market import FilterBy, SortBy
+from prediction_market_agent_tooling.markets.omen.omen_subgraph_handler import (
+    OmenSubgraphHandler,
+)
+
+from prediction_market_agent.agents.think_thoroughly_agent.models import (
+    PineconeMetadata,
+)
+from prediction_market_agent.db.pinecone_handler import PineconeHandler
+
+
+def main() -> None:
+    """Script for inserting all open markets into Pinecone (if not yet there)."""
+    sh = OmenSubgraphHandler()
+    open_markets = sh.get_omen_binary_markets_simple(
+        limit=sys.maxsize, filter_by=FilterBy.OPEN, sort_by=SortBy.NEWEST
```

Comment:
Hmm, good question. Should it be open markest only? Resolved ones should have correct answers.

Reply:
Per our discussion, refactoring this to use all markets (scripts + agent usages).